ocaml/xapi/pvs_proxy_control.ml

```diff
@@ -231,6 +232,18 @@ let start_proxy ~__context vif proxy =
             ignore(Client.Client.Message.create ~rpc ~session_id ~name ~priority ~cls:`PVS_proxy ~obj_uuid:proxy_uuid ~body));
         "unable to connect to PVS proxy daemon"
       | Api_errors.Server_error (code, args) when
+          code = Api_errors.pvs_site_missing_servers ->
+        let proxy_uuid = Db.PVS_proxy.get_uuid ~__context ~self:proxy in
+        let vif_uuid = Db.VIF.get_uuid ~__context ~self:vif in
+        let site_name_label = (Db.PVS_site.get_name_label ~__context ~self:(Db.PVS_proxy.get_site ~__context ~self:proxy)) in
```

Comment:
The first parenthesis is not necessary - the right hand side of a let can be any expression including a function call. Same one line down.